src/c++/perf_analyzer/infer_context.cc

```diff
@@ -32,9 +32,9 @@ void
 InferContext::Init()
 {
   if (shared_memory_type_ == SharedMemoryType::NO_SHARED_MEMORY) {
-    thread_stat_->status_ = PrepareInfer(this);
+    thread_stat_->status_ = PrepareInfer(infer_data_);
```

Comment:
Why does this pass in an object that's already accessible within the called function (data member)?